packages/editor-ui/src/components/canvas/elements/edges/utils/edgePath.ts

```diff
@@ -1,18 +1,19 @@
-import { getBezierPath, getSmoothStepPath, Position, type EdgeProps } from '@vue-flow/core';
+import type { ConnectionLineProps } from '@vue-flow/core';
+import { getBezierPath, getSmoothStepPath, Position } from '@vue-flow/core';
 
 const EDGE_PADDING_Y = 140;
 const EDGE_PADDING_Y_TOP = 80;
 const EDGE_BORDER_RADIUS = 8;
 const EDGE_OFFSET = 40;
 
-export function getCustomPath(props: EdgeProps) {
+export function getCustomPath(props: ConnectionLineProps) {
 	const { targetX, targetY, sourceX, sourceY, sourcePosition, targetPosition } = props;
 	const xDiff = targetX - sourceX;
 	const yDiff = targetY - sourceY;
 
 	// Connection is backwards and the source is on the right side
 	// -> We need to avoid overlapping the source node
-	if (xDiff < 0 && sourcePosition === Position.Right) {
+	if (xDiff < -16 && sourcePosition === Position.Right) {
```

Comment:
The `-16` is there to offset the size of the handle.

Reply:
Could you put 16 into a constant `HANDLE_SIZE`?